week-4/Homework/mandatory/4-groceries.js

```diff
@@ -28,8 +28,20 @@ Exercise 1:
   Then use console.log() to print out the list.
 */
 // Gather all week item names into this array
+
+// Solution 1
 let weeklyGroceriesToBuy = [];
+for (let key in weeklyMealPlan) {
+    let values = weeklyMealPlan[key]
+    for (let item of values) {
+        weeklyGroceriesToBuy.push(item)
```

Comment:
Impressive work here!
However, how about trying to do only one loop. and instead of looping inside an array, try to search for joining/concatenating arrays methods

Reply:
added solution2 by using concat